src/AppBundle/Controller/RegistrationController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Copyright 2018 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace AppBundle\Controller;
+
+use AppBundle\Tiqr\TiqrServiceInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Surfnet\GsspBundle\Service\RegistrationService;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class RegistrationController extends Controller
+{
+    private $registrationService;
+    private $tiqrService;
+
+    public function __construct(
+        RegistrationService $registrationService,
+        TiqrServiceInterface $tiqrService
+    ) {
+        $this->registrationService = $registrationService;
+        $this->tiqrService = $tiqrService;
+    }
+
+    /**
+     * Replace this example code with whatever you need.
```

Comment:
I would advice to remove the example phpdoc with a more meaningfull description. Or remove it altogether.